assets/data-port/export/export-job-state.js

```diff
@@ -0,0 +1,186 @@
+import apiFetch from '@wordpress/api-fetch';
+
+/**
+ * @typedef Job
+ *
+ * @property {Object}   status            Job status.
```

Comment:
I noticed the status, percentage, files, and error are at the same level in the `ExportProgressPage` component, but here we have the `status` separation. Is it for some purpose? Or should we make it similar?

Reply:
Renamed this to `job` everywhere after some refactoring.

Reply:
This is the backend response format (percentage and status inside a status object), but I unwrap it for the frontend to make it easier.